webapp/src/store/users.ts

```diff
@@ -21,6 +21,8 @@ export const fetchMe = createAsyncThunk(
     async () => client.getMe(),
 )
 
+export const versionProperty = 'version72MessageCanceled'
```

Comment:
For next version, this can be updated to new property name if you want to display...or set to ('') empty string to not display for this version.